packages/wrangler/src/pages/dev.ts

```diff
@@ -278,47 +295,39 @@ export const Handler = async ({
 		});
 	} else if (usingFunctions) {
 		// Try to use Functions
-		const outfile = join(tmpdir(), `./functionsWorker-${Math.random()}.mjs`);
-		scriptPath = outfile;
+		scriptPath = join(tmpdir(), `./functionsWorker-${Math.random()}.mjs`);
```

Comment:
Not part of your PR, but wouldnt it make more sense to have this idempotent by hashing someting dynamic and unique to the functionsWorker?

Reply:
It doesn't really matter what the name is as long as it is unlikely to collide with another concurrent build. This was the simplest solution.